android/key.properties

```diff
@@ -0,0 +1,4 @@
+keyAlias=foo
```

Comment:
Is this relevant? Or commited by mistake?

Reply:
Committed by mistake. I had it on my local .gitignore before it got squashed by pubspec update :)